### PR TITLE
Filter out libvirt events we don't care about

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

We only care about a few selected libvirt events like domain, network and pool lifecycle events as well as pool refreshes.

To limit the load this commit filters them at two levels:
  * the minion so that it doesn't send them out to the salt evens bus
  * the mgr_events engine to avoid them being stored in the DB in case some minions are still sending them.

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [X] **DONE**

## Test coverage
- No tests: already covered

- [X] **DONE**

## Links

Fixes https://github.com/SUSE/spacewalk/issues/19791

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: susemanager-utils/susemanager-sls/susemanager-sls.changes [hunk 1]
```diff
@@ -1,4 +1,4 @@
-- Improve _mgractionchains.conf logs
+- filter out libvirt engine events (bsc#1206146)
```
Comment (mcalmer) [inline susemanager-utils/susemanager-sls/susemanager-sls.changes:1]: seems you dropped a changelog

**Review by mcalmer — CHANGES_REQUESTED** [review]
please have a look at the changelog

**Review by rjmateus — COMMENTED** [review]
Besides MC comment, everything else looks good.

File: susemanager-utils/susemanager-sls/susemanager-sls.changes (hunk 1 above)
Comment (cbosdo) [reply susemanager-utils/susemanager-sls/susemanager-sls.changes:1]: oops

**Review by mcalmer — APPROVED** [review]

**Pull request merged**